packages/api/src/schema/schema.yaml

```diff
@@ -53,6 +53,14 @@ components:
         transcodedSegments:
           type: number
           example: 2
+        sourceSegmentsDuration:
+          type: number
+          example: 1
+          description: Duration of all the source segments, sec
+        transcodedSegmentsDuration:
```

Comment:
`sourceSegmentsDurationLastUpdated` and `transcodedSegmentsDurationLastUpdated` too, yeah?

Reply:
No, those two fields not saved to the db, they're used internally in the stream-info-app

Reply:
ah ok